src/translations/en.json

```diff
@@ -4596,8 +4596,8 @@
             "title": "YAML",
             "section": {
               "validation": {
-                "heading": "Configuration validation",
-                "introduction": "Validate your configuration if you recently made some changes to it and want to make sure that it is all valid.",
+                "heading": "Restart",
+                "introduction": "A basic validation of the configuration is automatically done before restarting. The basic validation ensures the YAML configuration doesn't have errors which will prevent Home Assistant or any integration from starting. It's also possible to only do the basic validation check without restarting.",
                 "check_config": "Check configuration",
                 "valid": "Configuration valid!",
```

Comment:
Should we change this to `Configuration will not prevent Home Assistant from starting!` 😝